lib/ash/filter/filter.ex

```diff
@@ -448,7 +448,11 @@ defmodule Ash.Filter do
       :error # not enough information
   """
   def get_filter(resource, id) do
+    multitenancy_attribute = Ash.Resource.Info.multitenancy_attribute(resource)
     primary_key = Ash.Resource.Info.primary_key(resource)
+
+    primary_key = Enum.reject(primary_key, fn key -> key == multitenancy_attribute end)
```

Comment:
We should do this only in the third branch of the conditional below, because this could result in an empty primary key which would cause problems here.